packages/dredd-transactions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dredd-transactions",
-  "version": "9.2.2",
+  "version": "10.0.0",
```

Comment:
Would that work as a dependency even though it is not released yet?

Reply:
Yes, that should work without issues as both packages will be published as a part of this release.